config/jail.conf

```diff
@@ -344,7 +344,7 @@ logpath = %(lighttpd_error_log)s
 [roundcube-auth]
 
 port     = http,https
-logpath  = /var/log/roundcube/userlogins
+logpath  = /var/log/roundcubemail/errors
```

Comment:
I guess original path was also present on some systems... so we shouldn't just blindly change it but rather add old value as roundcube_error_log to config/paths-common.conf,  use it here, and then customize for CentOS specifically I guess in config/paths-fedora.conf